opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/indicator-test.ts

```diff
@@ -178,7 +178,7 @@ describe('Indicator resolver standard behavior', () => {
 
     expect(queryResult.data?.indicator.decay_base_score).toBe(50);
     const startDecayDate: Date = queryResult.data?.indicator.decay_base_score_date;
-    expect(startDecayDate.getTime() / 1000).toBeCloseTo(new Date().getTime() / 1000, 0); // approximately now
+    expect(startDecayDate.getTime() / 10000).toBeCloseTo(new Date().getTime() / 10000, 0); // approximately now
```

Comment:
Why this change ?

Reply:
I just got a build fail on this PR because of this test that is a bit too agressive on date. So it's a flake, that I'm fixing in the way. I can do a separate PR but I prefer to have it soon in case it's distubing too much PR builds.

![Capture d'écran 2024-02-14 162749](https://github.com/OpenCTI-Platform/opencti/assets/3634942/c66bdb42-2fac-4a6b-9f83-643afeb7d709)